packages/component/src/components/page-list/operation-menu-items/export.tsx

```diff
@@ -123,7 +130,9 @@ const ExportToMarkdownMenuItem = ({
     if (!contentParserRef.current) {
       contentParserRef.current = new ContentParser(currentEditor.page);
     }
-    contentParserRef.current.exportMarkdown();
+    contentParserRef.current.exportMarkdown().catch(err => {
+      console.error(err);
+    });
     onSelect?.({ type: 'markdown' });
```

Comment:
should this `onSelect` be called in the `contentParserRef.current.exportMarkdown().then`?

Reply:
it will block the ui since export markdown is a heavy computing fn